package.json

```diff
@@ -24,15 +24,15 @@
   "author": "Wumke",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/wumke/react-native-exit-app/issues"
+    "url": "https://github.com/divvixyz/react-native-exit-app/issues"
   },
-  "homepage": "https://github.com/wumke/react-native-exit-app",
+  "homepage": "https://github.com/divvixyz/react-native-exit-app",
   "codegenConfig": {
     "name": "RNExitAppSpec",
     "type": "modules",
     "jsSrcsDir": ".",
     "android": {
-      "javaPackageName": "com.github.wumke.RNExitApp"
```

Comment:
I don't think we should rename this without renaming the actual folder used.

Reply:
Fixed in /b943dc7ebdb4c4acbe3264f19d1987d121c2c477.